scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -101,6 +101,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
             {
                 var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
                 AzurePublisherTool.PublishFile(hostBlob, file);
+                Console.WriteLine($"Publishing package {hostBlob} to Azure.");
```

Comment:
This is temporary to check the official builds, I will revert it after we get the official builds correctly publishing the package.